Toolchain/CMake/Scripts/DefaultPluginFunctions.cmake

```diff
@@ -0,0 +1,7 @@
+# Copyright MediaZ Teknoloji A.S. All Rights Reserved.
+
+function(nos_plugin_common current_dir out_target_dependencies)
+endfunction()
+
+function(nos_plugin_common_post_target_generation plugin_target)
```

Comment:
I think this should be named nos_plugin_on_post_target_generated. And we should pass a package name too, since we generate the target name, the developer might not know the target name to switch cases.